content/custom_exporter/Java/Trace.md

```diff
@@ -0,0 +1,281 @@
+---
+title: "Trace exporter"
+draft: false
+weight: 3
+---
+
+### Table of contents
+- [Introduction](#introduction)
+- [Implementation](#implementation)
+- [Runnable example](#runnable-example)
+- [Notes](#notes)
+- [References](#references)
+
+#### Introduction
+
+A trace exporter must extend the abstract class [SpanExporter.Handler](https://static.javadoc.io/io.opencensus/opencensus-api/0.15.0/io/opencensus/trace/export/SpanExporter.Handler.html) implementing the `export` method
+
+which for purposes of brevity is:
+
+```java
+import java.util.Collection;
+
+import io.opencensus.trace.export.SpanData;
+
+public void export(Collection<SpanData> spanDataList);
+```
+
+The sole method `export` will be used to process and translate a collection of [io.opencensus.trace.export.SpanData](https://static.javadoc.io/io.opencensus/opencensus-api/0.15.0/io/opencensus/trace/export/SpanData.html) to your desired trace backend's data.
+
+After an exporter is created, it must be registered with [SpanExporter.registerHandler](https://static.javadoc.io/io.opencensus/opencensus-api/0.15.0/io/opencensus/trace/export/SpanExporter.html#registerHandler-java.lang.String-io.opencensus.trace.export.SpanExporter.Handler-)
+
+```java
+SpanExporter.registerHandler(nameOfTheExporter, anInstanceOfTheExporter);
+```
+
+#### Implementation
+
+For example, let's make a custom trace exporter that will print span data to standard output.
+
+Inside file `src/main/java/oc/tutorials/CustomTraceExporter.java` we'll write the following code
```

Comment:
In Java we have a LogggingTraceExporter: https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/trace/logging/src/main/java/io/opencensus/exporter/trace/logging/LoggingTraceExporter.java. Maybe just reuse it as an example?

Reply:
I see. I wrote this though to be consistent with the Go exporter tutorial https://opencensus.io/custom_exporter/go/trace/ but the Logging Trace Exporter includes a bunch of annotations unrelated to the tutorial but also doesn't distinctly pick out the fields. They boil down to the same thing though :) plus also an exercise for me too to get content reviewed in from scratch instead of copied and pasted :)